crates/sui-protocol-config/src/lib.rs

```diff
@@ -3964,6 +3966,9 @@ impl ProtocolConfig {
                     // Reudce the frequency of checkpoint splitting under high TPS.
                     cfg.max_transactions_per_checkpoint = Some(20_000);
                 }
+                96 => {
```

Comment:
FYI v95 is still not in any release branch, although maybe by the time this lands it will be